src/pages/home/report/ReportActionItemThread.tsx

```diff
@@ -1,62 +1,61 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import {Text, View} from 'react-native';
-import avatarPropTypes from '@components/avatarPropTypes';
 import MultipleAvatars from '@components/MultipleAvatars';
 import PressableWithSecondaryInteraction from '@components/PressableWithSecondaryInteraction';
-import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
-import withWindowDimensions, {windowDimensionsPropTypes} from '@components/withWindowDimensions';
-import compose from '@libs/compose';
+import withWindowDimensions from '@components/withWindowDimensions';
+import {WindowDimensionsProps} from '@components/withWindowDimensions/types';
+import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@styles/useThemeStyles';
 import * as Report from '@userActions/Report';
 import CONST from '@src/CONST';
+import {Icon} from '@src/types/onyx/OnyxCommon';
 
-const propTypes = {
+type ReportActionItemThreadProps = WindowDimensionsProps & {
```

Comment:
I haven't found any `withWindowDimensions` HOC props used in this component, maybe we can get rid of it, wdyt?
```suggestion
type ReportActionItemThreadProps = {
```